src/ui/toolbar/MainToolBar.qml

```diff
@@ -121,6 +121,18 @@ Item {
                 }
             }
 
+            QGCToolBarButton {
+                id:                 photoGalleryButton
+                anchors.top:        parent.top
+                anchors.bottom:     parent.bottom
+                icon.source:        "/qmlimages/PhotoGallery.svg"
+                visible:            QGroundControl.corePlugin.showAdvancedUI
```

Comment:
I am not clear about this or the intended meaning of "advanced" -- can you clarify? Basically this follows the pattern around it.

Reply:
QGC has a concept of "Advanced Mode". When advance mode is enabled (not the default) some features that are otherwise hidden become visible. This is used by custom builds of QGC, where certain things should be hidden from the "casual" user (things like dangerous parameters and such). In this case, this new Image Gallery feature does not need to be hidden, therefore, you can remove the `visible` attribute altogether.

Reply:
removed visibility spec